Paymetheus.Decred/Wallet/StakeDifficultyProperties.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Paymetheus.Decred.Wallet
+{
+    public sealed class StakeDifficultyProperties
+    {
+        public int Height { get; set; }
```

Comment:
Be more descriptive about what height this is, whether it is the current height of this difficulty, or something else.